docs/index.md

```diff
@@ -84,14 +88,18 @@ with Webcam(fps=0.5) as webcam:
             frame.save_image("/tmp/latest-webcam-image.png", include_predictions=True) # (7)!
 ```
 
-1. Creates a LandingLens predictor object
-2. Here, you should set the endpoint_id you've got from the platform: <br/>![How to get endpoint ID](images/copy-endpoint-id.png "How to get endpoint ID").
-3. Here, you should set the API key you've got from the platform: <br/> ![How to get the API key](images/menu-api-key.png "How to get the API key").
-4. Resize the image to `width=512`, keeping the aspect ratio. This is useful to save some bandwidth when sending the image to LandingLens for prediction.
+1. Creates a LandingLens predictor object.
+2. Set the `endpoint_id` to the one from your deployed model in LandingLens: <br/>![How to get endpoint ID](images/copy-endpoint-id.png "How to get endpoint ID").
+3. Set the `api_key` to the one from your LandingLens organization: <br/> ![How to get the API key](images/menu-api-key.png "How to get the API key").
+4. Resize the image to `width=512`, keeping the aspect ratio. This is useful to save some bandwidth when sending the image to LandingLens for inference.
 5. Runs inference in the resized frame, and adds the predictions to the `Frame`.
-6. If a class names `coffee-mug` is found in the prediction
-7. Save the image with the predictions overlayed on top of the image.
+6. If the model predicts that an object with a class named `coffee-mug` is found...
+7. Then save the image with the predictions overlaid on the image.
+
+That's it! Now, with just a few lines of code, you can detect coffee mugs in front of your webcam.
 
-That's it. Now, with just a few lines of code, you detect coffee mugs passing by your webcam.
+Now, learn about the other ways to [acquire images](image-acquisition/image-acquisition.md), and how to process your images and [run inference on them](inferences/getting-started.md). For inspiration, check out our [Examples](examples.md).
+
+???+ note
 
-You can now check the [other image sources available](image-acquisition/image-acquisition.md), and how to process your images and [run prediction on them](inferences/getting-started.md). Check also our [Examples section](examples.md) for inspiration.
\ No newline at end of file
+    If you don't have a LandingLens account, create one [here](https://app.landing.ai/). Learn how to use LandingLens from our [Support Center]([https://support.landing.ai/docs/landinglens-workflow](https://support.landing.ai/landinglens/en)) and [Video Tutorial Library](https://support.landing.ai/docs/landinglens-workflow-2). Need help with specific use cases? Post your questions in our [Community](https://community.landing.ai/home).
```

Comment:
Can we move this note a bit above? Maybe right after the code snippet, or at least before the closing paragraph?

![image](https://github.com/landing-ai/landingai-python/assets/1213561/df44a679-f7be-4c25-9c5e-531f9c889478)